src/time.rs

```diff
@@ -272,12 +273,15 @@ impl IntoValue for chrono::DateTime<chrono::Utc> {
 impl IntoValue for chrono::DateTime<chrono::FixedOffset> {
     #[inline]
     fn into_value_with(self, ruby: &Ruby) -> Value {
-        use chrono::{DateTime, Utc};
-        let epoch = DateTime::<Utc>::UNIX_EPOCH.with_timezone(&self.timezone());
-        let delta = self.signed_duration_since(epoch);
-        ruby.time_nano_new(delta.num_seconds(), delta.subsec_nanos() as _)
+        use chrono::{DateTime, FixedOffset, Utc};
+        let delta = self.signed_duration_since(DateTime::<Utc>::UNIX_EPOCH);
+        let val = ruby
+            .time_nano_new(delta.num_seconds(), delta.subsec_nanos() as _)
+            .unwrap()
+            .as_value();
+        let offset: FixedOffset = self.timezone().into();
+        val.funcall("getlocal", (offset.local_minus_utc(),))
             .unwrap()
-            .as_value()
```

Comment:
There are a lot of `unwraps` here.  I was wondering whether there was the equivalent of a `TryIntoValue` that could be used instead.

Reply:
Usually it works out that Rust type is strict enough that it always converts to Ruby without error. Like even if you have to call a Ruby function that may error in other cases, the data from Rust will always be valid such that the error won't happen, and it's fine to just `unwrap`.

Like here `funcall` can error if you try to call a method on `nil`, but we know for sure `val` is a `Time` so it won't hit that error, and so on through all the possible errors.